websites/G/Goodreads/presence.ts

```diff
@@ -0,0 +1,35 @@
+const presence = new Presence({
+	clientId: "765622778317373440"
+}),
+elapsed = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo-1"
+	},
+	path = window.location.href.replace(window.location.origin, ''),
+	urlParamshere = new URLSearchParams(window.location.search);
+
+	if(path === '/'){
+		presenceData.details = "Browsing Home";
+	}else if(path.startsWith('/book/show/')){
+		bookname = path.replace('/book/show/', '').replace(/[0-9]+\./, '').replace(/[0-9]+-/, '').replace(/_/g, ' ').split('?')[0];

```

Comment:
```suggestion
		bookname = path
		               .replace('/book/show/', '')
		               .replace(/[0-9]+\./, '')
		               .replace(/[0-9]+-/, '')
		               .replace(/_/g, ' ')
                   .split('?')[0];
```

Better formatting?

Reply:
This format is good. It doesnt need to be in ur way.